styles/globals.css

```diff
@@ -14,3 +14,13 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* This code doesn't work and I don't know why. */
+
+pre {
+  margin: 25px 0 !important;
+}
+
+svg {
+  vertical-align: middle;
+}
```

Comment:
Ah, this is some left over file from Next's JSS global.

I'll move it over to the SC's global function.